download/upgradeRecipe/upgradeRecipe7.adoc

```diff
@@ -3338,6 +3338,19 @@ After in `benchmarkConfig.xml`:
     </plannerBenchmark>
 ----
 
+Furthermore, to continue to use that deprecated JavaScript configuration together
+with JDK 15 or later will have to add the following dependency to their runtime,
+on account of the OpenJDK project
```

Comment:
Who will have to add the dependency?

Reply:
=> "On the other hand, to continue to use that deprecated JavaScript expression configuration anyway:
code running on JDK 15 or later need to add the following dependency to its classpath,
on account of the OpenJDK project"